packages/dso-toolkit/components/Voorbeelden/20-Toepassingen/60-Helpcentrum/Startpagina.njk

```diff
@@ -0,0 +1,107 @@
+{% render '@header', {menu: menu, breadcrumbs: []} -%}
+<main>
+  <div class="row dso-banner dso-banner-implementation-specific-image" style="background-image: url('../../dummy/images/hands-on-trackpad.jpg')">
+    <!-- @Developers: Voeg dus geen inline styling toe maar genereer de betreffende styling in een class, zoals bv. `.dso-banner-implementation-specific image`. Inline styling doet het niet goed bij de Webrichtlijnen -->
```

Comment:
Gaat dit om maatwerk styling aan jullie kant?

Reply:
bedoel je die developers zin comment?  Die komt letterlijk uit de landingspagina ook. Dacht het is misschien belangrijk om die hier ook te plaatsen voor de zekerheid. https://dso-toolkit.nl/41.0.1/components/detail/landingspagina.html